qiita_db/study.py

```diff
@@ -199,6 +206,43 @@ def get_by_status(cls, status):
         return studies
 
     @classmethod
+    def get_info(cls, study_ids, info_cols=None):
```

Comment:
The name of this function is a bit confusing to me; I actually had to go and check how the instance method is called. Can we change it to something like `def get_info_by_study` and `study_ids` is optional (if not provided, it gives all the information for all the studies, probably useful from an admin point of view - that has access to everything).